ChangeLog

```diff
@@ -1,6 +1,8 @@
 CHANGES
 =======
 
+ADD: Print Uwsgi configuration options at docker startup
```

Comment:
No associated issue?

Reply:
Done in https://github.com/telefonicaid/orchestrator/pull/374/commits/624499101f14359eae87dc7f8066dc9008967f43